<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winUser</class>
 <widget class="QDialog" name="winUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dossier Utilisateur</string>
  </property>
  <widget class="QLabel" name="labelPicture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ressources/ressources.qrc">:/winPictures/openFolder-64.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dossiers Utilisateur</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFolder">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>39</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Desc. :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNameFolder">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDescFolder">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnOk">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>